integration_tests/require-all-modules/test.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+it('requires all native modules to check they all work', () => {
+  Object.keys(process.binding('natives'))
```

Comment:
If in a version of node `process.binding` changes to not return anything, this test will still pass but the sandboxing is broken. Can we change this to make sure it requires the right modules?

Reply:
Everything would be broken then, because we rely on `process.binding` already (e.g. `is_builtin_module` from `jest-resolve` 😄 

What I'm going to do is to check that the length is greater than the current amount of modules for Node 6, which should be good signaling.

Reply:
Yep, I'm fine with that. If they ever remove core modules our test will fail, but that's fine and something we'd want to know about anyway.